src/Forms/FormField.php

```diff
@@ -730,7 +730,11 @@ public function getAttributesHTML($attributes = null)
             if ($value === true) {
                 $parts[] = sprintf('%s="%s"', $name, $name);
             } else {
-                $parts[] = sprintf('%s="%s"', $name, Convert::raw2att($value));
+                $strValue = Convert::raw2att($value);
+                if (!is_string($strValue)) {
+                    $strValue = json_encode($strValue);
```

Comment:
We could use `Convert::raw2json()` here

Reply:
what is the added value of calling raw2json since it does the same thing? If I scan the codebase, I see a lot of direct calls to json_encode already.

Reply:
Consistency of dogfooding our API really, otherwise what's the point in having it 😄